frontend/src/components/ParticipantInfo.js

```diff
@@ -0,0 +1,544 @@
+import { rootStoreContext } from "../stores/RootStore"
+import "../scss/participant-search.scss"
+import FormGroup from "@material-ui/core/FormGroup"
+import FormControl from "@material-ui/core/FormControl"
+import InputLabel from "@material-ui/core/InputLabel"
+import Input from "@material-ui/core/Input"
+import Typography from "@material-ui/core/Typography"
+import Grid from "@material-ui/core/Grid"
+import TextField from "@material-ui/core/TextField"
+import Container from "@material-ui/core/Container"
+import MenuItem from "@material-ui/core/MenuItem"
+import Select from "@material-ui/core/Select"
+import { makeStyles } from "@material-ui/core/styles"
+import Radio from "@material-ui/core/Radio"
+import RadioGroup from "@material-ui/core/RadioGroup"
+import FormControlLabel from "@material-ui/core/FormControlLabel"
+import FormLabel from "@material-ui/core/FormLabel"
+import Button from "@material-ui/core/Button"
+import { observer } from "mobx-react-lite"
+import { useHistory } from "react-router-dom"
+// import { toJS } from "mobx"
+
+const useStyles = makeStyles(theme => ({
+  paper: {
+    position: "absolute",
+    width: 400,
+    backgroundColor: theme.palette.background.paper,
+    border: "2px solid #000",
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+  },
+  card: {
+    float: "right",
+    width: "25%",
+    marginRight: "35px",
+  },
+  hidden: {
+    display: "none",
+  },
+  bullet: {
+    display: "inline-block",
+    margin: "0 2px",
+    transform: "scale(0.8)",
+  },
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    marginBottom: 12,
+  },
+  root: {
+    display: "flex",
+    flexWrap: "wrap",
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 200,
+    maxWidth: 300,
+  },
+  chips: {
+    display: "flex",
+    flexWrap: "wrap",
+  },
+  chip: {
+    margin: 2,
+  },
+  noLabel: {
+    marginTop: theme.spacing(3),
+  },
+}))
+
+const ParticipantInfo = observer(() => {
+  const rootStore = useContext(rootStoreContext)
+  const participantStore = rootStore.ParticipantStore
+
+  const [open, setOpen] = React.useState("")
+  const [id, setId] = React.useState(0)
+  const [lastFourSSN, setSSN] = React.useState(0)
+  const [startDate, setStartDate] = React.useState("")
+  const [insurer, setInsurer] = React.useState("")
+  const [firstName, setFirstName] = React.useState("")
+  const [lastName, setLastName] = React.useState("")
+  const [dateOfBirth, setDateOfBirth] = React.useState("")
+  const [ppId, setPPId] = React.useState("")
+  const [race, setRace] = React.useState("")
+  const [gender, setGender] = React.useState("")
+  const [hasInsurance, setHasInsurance] = React.useState(false)
+  const [insuranceType, setInsuranceType] = React.useState("")
+  const [program, setProgram] = React.useState("")
+  const [service, setService] = React.useState("")
+  const [priority, setPriority] = React.useState("")
+  const [note, setNote] = React.useState("")
```

Comment:
What do you think about using one object that contains all of these properties? For instance:

```javascript
const [participant, setParticipant] = React.useState({})
```

And then from the Input component we can make the `onChange` function look like the following:

```javascript
<Input
  onChange={e => setParticipant(prevState => { ...prevState, thisProperty: e.target.value })}
  ...
/>
```

Reply:
I love it and originally had that idea in mind but just opted for this right now for demo purposes. I'd much rather prefer one object in local state than the approach I've pushed